classes/TaskStatusAction.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Класс описывает все состояния задания
+ * и все возможные действия с ним
+ * @property const CLIENT
+ * @property const DOER
+ * @property const STATUS_NEW
+ * @property const STATUS_UNDO
+ * @property const STATUS_INWORK
+ * @property const STATUS_REFUSAL
+ * @property const STATUS_FINISH
+ * @property const ACTION_CREATE
+ * @property const ACTION_CANCEL
+ * @property const ACTION_RESPOND
+ * @property const ACTION_DO
+ * @property const ACTION_REFUSE
+ * @property const ACTION_FINISH
+ * @property int $userId
+ * @property int $taskId
+ * @property string $userRole
+ * @property string $actualStatus
+ * @property array $statuses
+ * @property array $actions
+ * @property string $status
+ * @property string $action
+ * @method getStatus()
+ * @method getAction()
+ * @method getActualStatus(string $action)
+ * @method getClientAllowedAction(string $status)
+ * @method getDoerAllowedAction(string $status)
+ */
+
+class TaskStatusAction
```

Comment:
```suggestion
class TaskStatusAction
```
между phpdoc и элементом, к которму этот комментарий относится, не должно быть пустой строки